clients/drcachesim/docs/drcachesim.dox.in

```diff
@@ -1146,6 +1146,8 @@ $ bin64/drrun -t drcachesim -indir drmemtrace.app.pid.xxxx.dir/
 
 The direct results of the \p -offline run are raw, compacted files, stored
 in a \p raw/ subdirectory of the \p drmemtrace.app.pid.xxxx.dir directory.
+The contents of this directory are internal to the tool.  Do not alter, add,
+or delete files here.
 The \p -indir option both converts the data to a canonical trace form and
 passes the resulting data to the cache simulator.  The canonical trace data
 is stored by \p -indir in a \p trace/ subdirectory inside the \p
```

Comment:
The same warning applies to the trace/ subdir too.  (Or clarify the above to explicitly state it's talking about the outer dir and not the raw/ subdir.)

Reply:
How about say that the contents of drmemtrace.app.pid.xxxx.dir, and all its subdirectories, are internal to the tool ?

Reply:
SGTM